app/assets/stylesheets/components/auth-form.scss

```diff
@@ -1,26 +1,27 @@
+@import "base/colors";
 
 .auth-form {
+  background: #f8fbfd;
+  border: 1px solid $field-border;
+  border-radius: 5px;
   display: flex;
   flex-direction: column;
   margin: 30px auto;
-  background: #f8fbfd;
-  border: 1px solid #dae1e3;
-  border-radius: 5px;
   max-width: 450px;
   padding: 45px;
+  position: relative;
   text-align: left;
   width: 100%;
-  position: relative;
 
   .avatar-placeholder {
-    height: 80px;
-    width: 80px;
+    background-color: #C6DAE3;
+    border: 3px solid white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.